### ISSUE TITLE
Coverage.py measures itself on Precise Pandolin

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#214]
From #python today, another report of this.  Here's the "coverage debug sys":

```
$ python-coverage debug sys
-- sys ----------------------------------------
        version: 3.4
       coverage: /usr/lib/python2.7/dist-packages/coverage/__init__.pyc
      cover_dir: /usr/lib/python2.7/dist-packages/coverage
     pylib_dirs:
                 /usr/lib/python2.7
         tracer: Tracer
      data_path: /home/dev/vminstaller/.coverage
         python: 2.7.3 (default, Aug  1 2012, 05:14:39) [GCC 4.6.3]
       platform: Linux-3.2.0-33-generic-x86_64-with-Ubuntu-12.04-precise
            cwd: /home/dev/vminstaller
           path:
                 /usr/bin
                 /home/cr3/lib/python
                 /usr/lib/python2.7
                 /usr/lib/python2.7/plat-linux2
                 /usr/lib/python2.7/lib-tk
                 /usr/lib/python2.7/lib-old
                 /usr/lib/python2.7/lib-dynload
                 /usr/local/lib/python2.7/dist-packages
                 /usr/lib/python2.7/dist-packages
                 /usr/lib/python2.7/dist-packages/PIL
                 /usr/lib/python2.7/dist-packages/gst-0.10
                 /usr/lib/python2.7/dist-packages/gtk-2.0
                 /usr/lib/pymodules/python2.7
                 /usr/lib/python2.7/dist-packages/ubuntu-sso-client
                 /usr/lib/python2.7/dist-packages/ubuntuone-client
                 /usr/lib/python2.7/dist-packages/ubuntuone-control-panel
                 /usr/lib/python2.7/dist-packages/ubuntuone-couch
                 /usr/lib/python2.7/dist-packages/ubuntuone-installer
                 /usr/lib/python2.7/dist-packages/ubuntuone-storage-protocol
    environment:
                 PYTHONPATH = /home/cr3/lib/python
```

And here's the coverage report:

```
/usr/share/pyshared/coverage/collector         132    127     4%   3-229, 236-244, 248-292
/usr/share/pyshared/coverage/control           236    235     1%   3-355, 358-624
/usr/share/pyshared/coverage/execfile           35     14    60%   3-17, 42-43, 48, 58-65
test                                            33      6    82%   21-24, 27, 48
vminstaller/__init__                             2      0   100%
vminstaller/command                             12      7    42%   17-24
vminstaller/disk                                80      0   100%
vminstaller/initrd                              70     12    83%   102-119, 128, 135, 147, 154
vminstaller/iso/__init__                         0      0   100%
vminstaller/iso/inode                          157      3    98%   110, 161, 202
vminstaller/iso/tests/__init__                   0      0   100%
vminstaller/iso/tests/test_inode               228      0   100%
vminstaller/monitor                             27      0   100%
vminstaller/scripts/__init__                     0      0   100%
vminstaller/scripts/application                 39      0   100%
vminstaller/scripts/logger                      52     16    69%   33-44, 48-59, 71-72, 89
vminstaller/scripts/tests/__init__               0      0   100%
vminstaller/scripts/tests/test_application      82      0   100%
vminstaller/scripts/tests/test_logger           37      0   100%
vminstaller/testing/__init__                     0      0   100%
vminstaller/testing/temp                        54      0   100%
vminstaller/testing/tests/__init__               0      0   100%
vminstaller/testing/tests/test_temp             81      0   100%
vminstaller/testing/tests/test_unique           21      0   100%
vminstaller/testing/unique                      19      0   100%
vminstaller/tests/__init__                      25      4    84%   37-41
vminstaller/tests/test_disk                    147      0   100%
vminstaller/tests/test_initrd                   71      0   100%
vminstaller/tests/test_monitor                  68      0   100%
--------------------------------------------------------------------------
TOTAL                                    
```

veloutin reports that the files in /...2.7/dist-packages/ are symlinks to those in /..pyshared/

----------------------------------------
- Bitbucket: https://bitbucket.org/ned/coveragepy/issue/214

**Comment by nedbat** [comment]
Turns out coverage is installed in an odd (?) way:

```
ned@ned-precise:/usr/lib/python2.7/dist-packages/coverage$ ls -l
total 212
lrwxrwxrwx 1 root root    47 Apr 23  2012 annotate.py -> ../../../../share/pyshared/coverage/annotate.py
-rw-r--r-- 1 root root  3198 Nov 22 21:16 annotate.pyc
lrwxrwxrwx 1 root root    47 Apr 23  2012 backward.py -> ../../../../share/pyshared/coverage/backward.py
-rw-r--r-- 1 root root  1590 Nov 22 21:16 backward.pyc
lrwxrwxrwx 1 root root    47 Apr 23  2012 bytecode.py -> ../../../../share/pyshared/coverage/bytecode.py
-rw-r--r-- 1 root root  3338 Nov 22 21:16 bytecode.pyc
```

**Comment by nedbat** [comment]
This is fixed in <<changeset [aae421baacfb28d240010c7ae7341698c0559b8a (bb)](https://bitbucket.org/ned/coveragepy/commits/aae421baacfb28d240010c7ae7341698c0559b8a)>>.

**Comment by nedbat** [comment]
BTW, Note this ticket was cloned from #74.
